tooling/Dockerfile

```diff
@@ -1,16 +1,24 @@
-FROM fedora:30 AS kafkacat
+FROM fedora:34 AS kcat
 RUN dnf -y install gcc which gcc-c++ wget make git cmake
-ENV KAFKACAT_VERSION=1.5.0
+ENV KCAT_VERSION=1.5.0
 RUN dnf -y install cyrus-sasl-devel libcurl-devel libzstd-devel
-RUN git clone https://github.com/edenhill/kafkacat -b $KAFKACAT_VERSION --single-branch && \
-    cd kafkacat && \
+RUN git clone https://github.com/edenhill/kcat -b $KCAT_VERSION --single-branch && \
+    cd kcat && \
     ./bootstrap.sh
 
-FROM fedora:30
+FROM fedora:34
 RUN dnf -y install jq httpie mycli python3-pip gcc redhat-rpm-config python-devel libpq-devel &&\
     pip install pgcli &&\
     dnf clean all
-COPY --from=kafkacat /kafkacat/kafkacat /usr/bin/kafkacat
+COPY --from=kcat /kcat /usr/bin/kcat
+COPY --from=kcat /kcat /usr/bin/kafkacat
+RUN ln -s /usr/bin/kcat/ /usr/bin/kafkacat/
+
+FROM fedora:34
+RUN curl -SL https://github.com/kcctl/kcctl/releases/download/1.0.0-early-access/kcctl-1.0.0-early-access-linux-x86_64.tar.gz | tar -zx && \
+    mv kcctl-1.0.0-SNAPSHOT-linux-x86_64/bin/kcctl /usr/bin && \
+    . kcctl-1.0.0-SNAPSHOT-linux-x86_64/kcctl_completion && \
```

Comment:
Does sourcing the completion file here take any effect actually? Or would this rather have to go into the bashrc of the container's user?

Note it's only semi-useful anyways, as most of the times, usage of the tooling commands happens interactively from the host, so any container-side completion won't help. OTOH it would be nice to have it there in case someone simply opens a shell for the tooling container. Having completion support for that case wold be nice, but I don't think it'd work in the currently suggested way.

Curious about your thoughts too, @jpechane.

Reply:
> Does sourcing the completion file here take any effect actually? Or would this rather have to go into the bashrc of the container's user?

I agree with this point. What should be done then @gunnarmorling?

Reply:
Let's simply remove it for now.